### DISCUSSION TITLE
更新包下载完成，然后呢？

### DISCUSSION THREAD

**Discussion by Rangtian** [leiurayer/downkyi#734] [Q&A] [answered]
请教各位大佬：

更新包下载完成后，
既没有看到确认安装更新的弹窗，
又没能成功更新。
关闭程序后再次打开，
依旧是1.5.7，依旧提示更新。

![image](https://user-images.githubusercontent.com/22640407/235464335-358fd293-5451-42dc-86bb-78589b1bb135.png)

![image](https://user-images.githubusercontent.com/22640407/235464365-7a20a203-430e-4571-a5e7-d2aef01a6e8a.png)

**Comment by leiurayer** [comment, marked as answer]
在程序目录下，有下载的更新包，手动解压

**Reply by Rangtian** [reply]
谢谢你
